core/src/test/java/org/axonframework/eventsourcing/eventstore/jpa/JpaEventStorageEngineTest.java

```diff
@@ -89,11 +88,6 @@ public void setUp() throws SQLException {
         entityManager.clear();
     }
 
-    @After
```

Comment:
Why was this removed?
This test sets the clock to a fixed time a couple of times. This was to ensure the "regular" clock was reinstated after these tests.

Reply:
Take a look at `EventStorageEngineTest#tearDown` method.

Reply:
👍